File: Python/39.py
import os
aresta=0
aresta1=0
aresta2=0
vol=0
volexist=0
print("\n\nEste algoritmo calcula o combustivel existente em um reservatorio,que esta a 60% de sua capacidade.")
aresta=float(input("\n\nDigite o valor da aresta desse reservatorio: "))
aresta1=float(input("\n\nDigite o valor da aresta desse reservatorio: "))
aresta2=float(input("\n\nDigite o valor da aresta desse reservatorio: "))
vol=(aresta*aresta1)*aresta2
volexist=(60*vol)/100
print(f"\n\nEste é o valor do combustivel existente nesse reservatorio : {volexist}")
print("\n\nObrigado por utilizar nosso programa")







os.system("pause")

File: C/39.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(void){
	setlocale(LC_ALL, "Portuguese");
	float vol;

	printf("Esse programa calcula o volume de agua dentro de um reservatorio!\n\n");
	
	printf("\n\nCapacidade total do reservatorio: ");
	scanf("%f",&vol);
	
	vol=(vol*60)/100;
	printf("\n\nO volume existente no reservatorio é de %.1f!",vol);
	printf("\n\nObrigado por usar nosso sistema!\n\n");
	system("pause");
	return 0;
}


